.openshift-ci/jobs/integration-tests/run-benchmarks.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BRANCH="$(jq -r '.extra_refs[0].base_ref' <(echo "$JOB_SPEC"))"
+
+function run_benchmarks() {
+    make -C ansible BUILD_TYPE=ci benchmarks
+}
+
+function save_artifacts() {
+    cp "integration-tests/perf.json" "${ARTIFACT_DIR}/${JOB_NAME_SAFE}-perf.json"
+    cp -r "integration-tests/container-logs/" "${ARTIFACT_DIR}"
+
+    if [[ -d "integration-tests/performance-logs" ]]; then
+        cp -r "integration-tests/performance-logs" "${ARITFACT_DIR}"
+    fi
+}
+
+run_benchmarks || exit_code=$?
```

Comment:
Define `exit_code` as 0 just to be double safe it is defined if the benchmark works correctly.
```suggestion
exit_code=0
run_benchmarks || exit_code=$?
```